Makefile

```diff
@@ -1,29 +1,14 @@
 NAME    := mercury
 SRC_EXT := bz2
 SOURCE   = https://github.com/mercury-hpc/$(NAME)/releases/download/v$(VERSION)/$(NAME)-$(VERSION).tar.$(SRC_EXT)
-#PATCH1 is reversed for RPMS.
-PATCH1  := https://github.com/mercury-hpc/mercury/compare/c68870ffc0409c29eece5ba036c6efd3c22cee41^...v1.0.1.patch
-ID_LIKE1=$(shell . /etc/os-release; echo $$ID_LIKE)
-ifeq ($(ID_LIKE1),debian)
-# Debian can not use a reverse patch
-PATCHES := $(shell rm -rf _topdir/BUILD) 0013-Update-copyright-date.patch \
-	0012-Update-README-for-req-gcc-version-w-stdatomic.h.patch \
-	0011-Update-CHANGELOG.patch \
-	0010-HG-Fix-HG_Reset-to-reset-NA-resources-upon-NA-class-.patch \
-	0009-HG-Util-move-inline-functions-for-get-set-and-export.patch \
-	0008-Travis-update-build-script-to-OFI-1.7.0.patch \
-	0007-Travis-update-build-script-to-OFI-1.7.0rc3.patch \
-	0006-NA-SM-remove-page-size-check-that-would-prevent-to-r.patch \
-	0005-Update-CMake-policy-for-CMake-3.12-and-above.patch \
-	0004-HG-Core-fix-potential-race-when-forcing-a-handle-to-.patch \
-	0003-Update-travis-CI-to-use-latest-libfabric-CMake-MPICH.patch \
-	0002-NA-OFI-fix-cancelation-of-operations-that-cannot-be-.patch \
-	0001-HG-fix-cancelation-of-HG-operations-fix-267.patch
-else
-PATCHES := c68870ffc0409c29eece5ba036c6efd3c22cee41^...v1.0.1.patch
-endif
+PATCHES := v1.0.1...cc0807e8377e129945834d292be21a6667a8cbb3.patch                                   \
+	       cc0807e8377e129945834d292be21a6667a8cbb3...f0b9f992793be46f1c6ae47b30d1c3ccb525cfbf.patch
 
-c68870ffc0409c29eece5ba036c6efd3c22cee41^...v1.0.1.patch:
-	curl -f -L -O '$(PATCH1)'
+el7_PR_REPOS  := libpsm2@PR-1
+PR_REPOS      := libfabric@PR-16
```

Comment:
Don't we want to get these PRs landed first?

Reply:
Oh, good catch.  Those shouldn't be in there.

Reply:
Fixed.  Those updates for PSM2 will be in the next round.